src/plugins/intel_cpu/src/nodes/generate_proposals.cpp

```diff
@@ -436,8 +324,23 @@ void GenerateProposals::execute(dnnl::stream strm) {
                               });
 
             unpack_boxes(reinterpret_cast<float *>(&proposals_[0]), &unpacked_boxes[0], &is_dead[0], pre_nms_topn);
-            nms_cpu(pre_nms_topn, &is_dead[0], &unpacked_boxes[0], &roi_indices_[0], &num_rois, 0,
+
```

Comment:
I propose to approach parallelization strategy in the upcoming PR dedicated to GenerateProposals operation since it can affect other kernels from GenerateProposals op. @lohika-denis-kotov could you please take a look at the comment above?